ui/src/components/Log/index.jsx

```diff
@@ -19,89 +19,105 @@ const LINE_NUMBER_MATCH = /L(\d+)-?(\d+)?/;
+                  ...filterStyles['&.active'],
+                  fill: theme.palette.text.active,
+                },
+                '&.inactive': {
+                  fill: theme.palette.text.disabled,
+                },
               },
-              '&.inactive': {
-                fill: theme.palette.text.disabled,
+              '& > .react-lazylog-searchbar-matches': {
+                '&.active': {
+                  color: theme.palette.text.active,
+                },
+                '&.inactive': {
+                  color: theme.palette.text.disabled,
+                },
               },
-            },
-            '& > .react-lazylog-searchbar-matches': {
-              '&.active': {
-                color: theme.palette.text.active,
+            }
+          : {
+              '& > .react-lazylog-searchbar-input': {
+                height: theme.spacing.triple,
               },
-              '&.inactive': {
-                color: theme.palette.text.disabled,
+              '& > .react-lazylog-searchbar-filter': {
+                ...filterStyles,
               },
-            },
-          }
-        : {
-            '& > .react-lazylog-searchbar-input': {
-              height: theme.spacing.triple,
-            },
-          }),
+            }),
+      },
     },
-  },
-  highlight: {
-    backgroundColor: `${theme.palette.action.selected} !important`,
-  },
-  line: {
-    '& > a': {
-      transition: theme.transitions.create('color'),
-      color: theme.palette.text.inactive,
+    highlight: {
+      backgroundColor: `${theme.palette.action.selected} !important`,
     },
-    '&:hover > a': {
-      color: `${theme.palette.text.primary} !important`,
+    line: {
+      '& > a': {
+        transition: theme.transitions.create('color'),
+        color: theme.palette.text.inactive,
+      },
+      '&:hover > a': {
+        color: `${theme.palette.text.primary} !important`,
+      },
+      '&$highlight > a': {
+        color: theme.palette.text.primary,
+      },
+      '&:hover': {
+        backgroundColor: `${theme.palette.action.hover} !important`,
+      },
     },
-    '&$highlight > a': {
-      color: theme.palette.text.primary,
+    logActions: {
+      position: 'absolute',
+      top: 6,
+      right: 340,
     },
-    '&:hover': {
-      backgroundColor: `${theme.palette.action.hover} !important`,
+    followButtonFollowing: {
+      ...theme.mixins.successIcon,
     },
-  },
-  logActions: {
-    position: 'absolute',
-    top: 6,
-    right: 340,
-  },
-  followButtonFollowing: {
-    ...theme.mixins.successIcon,
-  },
-  fabIcon: {
-    ...theme.mixins.fabIcon,
-  },
-  logSpeedDial: {
-    ...theme.mixins.fab,
-    ...theme.mixins.actionButton,
-    bottom: theme.spacing.triple,
-  },
-}))
+    fabIcon: {
+      ...theme.mixins.fabIcon,
+    },
+    logSpeedDial: {
+      ...theme.mixins.fab,
+      ...theme.mixins.actionButton,
+      bottom: theme.spacing.triple,
+    },
+  };
+})
```

Comment:
Any chance you could give an overview of the changes you made? The diff is a bit hard to follow for some reason 😅

Reply:
Sure. I added a `filterStyles` object [here](https://github.com/taskcluster/taskcluster/pull/1183/files#diff-6af56ca36cff0662454eca418e5191b6R24) for the active filter styles in both dark and light themes. And I updated the follow log button styles [here](https://github.com/taskcluster/taskcluster/pull/1183/files#diff-6af56ca36cff0662454eca418e5191b6R110). 

Does that help?